arrays.md

```diff
@@ -764,21 +762,15 @@ functions:
 
 The [Matlisp](https://github.com/matlisp/matlisp) scientific
 computation library provides high performance operations on arrays,
-including wrappers around BLAS and LAPACK functions. It is not yet on
-Quicklisp, but can be installed with the following commands if you
-already have Quicklisp installed:
-~~~bash
-$ cd ~/quicklisp/local-projects
-$ git clone https://github.com/matlisp/matlisp.git
-~~~
-
-Then loaded using quicklisp:
+including wrappers around BLAS and LAPACK functions. 
+It can be loaded using quicklisp:
 
 ~~~lisp
 * (ql:quickload :matlisp)
 ~~~
 
-To avoid typing `matlisp:` in front of each symbol, you can use
+The nickname for `matlisp` is `m`, but to avoid typing `matlisp:` or
+`m:` in front of each symbol, you can use
 the package or for an interactive session run:
 ~~~lisp
 * (in-package :matlisp)
```

Comment:
what about also sometimes suggesting `use-package` ? (also re-showing a package definition using some libraries cannot hurt).

Reply:
Yes, good idea. I'll add something on using the packages.